src/UserManagerSettings.ts

```diff
@@ -15,10 +16,11 @@ export interface UserManagerSettings extends OidcClientSettings {
     popup_redirect_uri?: string;
     popup_post_logout_redirect_uri?: string;
     /**
-     * The features parameter to window.open for the popup signin window.
-     * (default: "location=no,toolbar=no,width=500,height=500,left=100,top=100;")
+     * The features parameter to window.open for the popup signin window. By default, the popup is
+     * placed centered in front of the window opener.
+     * (default: "location=no,toolbar=no,width=576,height=640")
```

Comment:
comment needs to be adapted as it is an object now:
(default: { location: "no", .... })